Loader/SymfonyFixturesLoader.php

```diff
@@ -72,6 +78,47 @@ protected function createFixture($class)
         return $this->loadedFixtures[$class];
     }
 
+    /**
+     * Returns the array of data fixtures to execute.
+     *
+     * @param string $set
+     *
+     * @return array $fixtures
+     */
+    public function getFixtures($set = null)
+    {
+        $fixtures = parent::getFixtures();
```

Comment:
Be aware that this array already has the fixture classes as keys. So you do not actually need to call `get_class` again on each fixture, because that is already stored in the array key.

I know, it is a micro optimisation, but I like code that does not do extra tasks that is not needed.